src/caselawclient/Client.py

```diff
@@ -634,12 +634,14 @@ def checkout_judgment(
         judgment_uri: DocumentURIString,
         annotation: str = "",
         expires_at_midnight: bool = False,
+        timeout_seconds: int = -1,
```

Comment:
My only thought would be if an infinite lock is the right default behaviour, but I guess it's easy enough to fix if we discover the answer is "no".

Reply:
That's fair; it is the existing default behaviour.